schema/swagger.yml

```diff
@@ -2610,7 +2603,7 @@ definitions:
       signatures:
         type: array
         items:
-          $ref: '#/definitions/Signature'
+          type: object
```

Comment:
@nazarhussain That's correct. Otherwise if a single signature in the batch/array doesn't match the schema then the whole batch will fail validation which we don't want. Now it only enforces the array min and max lengths.